### DISCUSSION TITLE
Kitty-Symbols

### DISCUSSION THREAD

**Discussion by hubkae** [0x6d69636b/windows_hardening#79] [General]
Hi, how do i get the Symbols working ? The Output just shows me strange Chars ...

Thanks

**Comment by 0x6d69636b** [comment]
Hi hubkae,

You need a terminal that supports Unicode character like [Windows Terminal](https://www.microsoft.com/en-US/p/windows-terminal/9n0dx20hk701). Sadly, the default PowerShell shell does not support the kitty emojis. :(

**Comment by hubkae** [comment]
Hi,

Thank You !! Unfortunately there seems to be no Windows Terminal Version for Server 2019 available at the Moment ... :(

**Reply by 0x6d69636b** [reply]
Didi you see this issue? It might be possible :) https://github.com/microsoft/terminal/issues/1483
